cli/smartcontract/smart_contract.go

```diff
@@ -771,7 +771,7 @@ func contractDeploy(ctx *cli.Context) error {
 		return err
 	}
 
-	mgmtHash, err := c.GetNativeContractHash("Neo Contract Management")
+	mgmtHash, err := c.GetNativeContractHash("ManagementContract")
```

Comment:
Again, what about exporting these from `native`?

Reply:
This requires importing `native` in target packages, I'm not sure this is appropriate (clients messing with chain's internals), these names can be seen as a part of API and clients need to know them, they also are supposed to be immutable for chain's lifetime. We can of course define them client-side or move to a separate package (like `interopnames`).

Reply:
Moved names to `nativenames`.